contents/spotlight/startup-puzzle.md

```diff
@@ -0,0 +1,79 @@
+
+## 1. Collect both qualitative and quantitative data
+
+An ICP is, fundamentally, a snapshot of your ideal customer based on a gestalt of many customers. In order to define your ICP you therefore need to start by speaking to your existing users directly and building an intuitive understanding of their needs.
+
+Once you’ve done that, says Scott, you need to pair that intuitive understanding with quantitative data so you can understand if these behaviors exist in isolation or at scale. 
+
+“We’re constantly standing next to founders who are just starting out and I do a healthy amount of interviews,” says Scott. “But it’s one thing for a customer to say they want, need, or plan to do X, Y, or Z — and another thing if they actually do it. That’s where quantitative data is useful. You need both to create your ICP.”
+
+> **Putting it into practice:** It can help to [define a small number of reference customers](/founders/creating-ideal-customer-profile#5-reference-customers). Spend time with them to gather subjective data, while using a tool such as [PostHog](/startups) to collect hard facts.
+
+## 2. Understand how your users _feel_
+
+Armed with this qualitative and quantitative data, you can start to establish your value proposition.
+
+For Puzzle, this was critical for differentiating them from other products in the market. Every company needs an accounting platform, but not all founders feel the same about this need — and by focusing on users’ unaddressed needs Puzzle can identify a strong, unique position for the product.
+
+“Founders don’t sign up to Puzzle because they just can’t wait to do their accounting,” says Scott. “They sign up because they know they need to be tax compliant and to be ready for investor reporting...We learned that they care about efficiency, accuracy and, as they generate more revenue, delaying hires in the finance department.”
+
+All of this is reflected in Puzzle’s online messaging, with language which directly addresses founders ahead of accountants and emphasizes how the product integrates simply with other tools which founders already use. 
+
+> **Putting it into practice:** [We advise having an approximate structure for user interviews](/blog/how-to-work-out-what-users-need), but leave room to flex. If users go off on a tangent during an interview that could give you vital [insights to their actual feelings](/handbook/product/user-feedback)!
+
+## 3. Build pricing around user segments
+
+Next, you can delve into the quantitative data by looking at how users can be segmented into different groups based on criteria such as size or industry. Puzzle used this information to structure its pricing model, for example. 
+
+“On [our pricing page](https://puzzle.io/pricing) we make this pretty clear,” says Scott. “We have a plan for formation-stage founders, a plan for startups who have raised a little funding, and a scale plan for companies which are generating revenue.
```

Comment:
“On [our pricing page](https://puzzle.io/pricing) we make this pretty clear,” says Scott. “We have a plan for formation-stage founders, a plan for startups who have raised a little funding, and an automation plan for companies which are a little more mature and prioritize efficiency, accuracy, and not waiting weeks to close their books.

Reply:
```suggestion
“On [our pricing page](https://puzzle.io/pricing) we make this pretty clear,” says Scott. “We have a plan for formation-stage founders, a plan for startups who have raised a little funding, and an automation plan for companies which are a little more mature and prioritize efficiency, accuracy, and not waiting weeks to close their books."
```